client/components/App/App.css

```diff
@@ -352,6 +356,64 @@ img {
     max-width: 100%;
 }
 
+.get-involved {
+    background-color: #f8f8f8;
+    padding: 5em 0;
+}
+
+.get-involved .container {
+    background: none;
+}
+
+.get-involved .container h2 {
+    margin: 0 0 1em 0;
+    text-align: center;
+    font-size: 2em;
+    text-transform: uppercase;
+    font-weight: bold;
+}
+
+.get-involved .resources {
+    display: grid;
+    grid-template-columns: 19% 19% 19% 19% 19%;
+    column-gap: 1.25%;
+}
+
+.get-involved .resources article {
+    background: white;
+    padding: 2em 1em;
+    box-shadow: 0px 6px 11px 0px rgba(0, 0, 0, 0.15);
+    border-radius: 5px;
+    text-align: center;
+}
+
+.get-involved .resources img {
+    width: 100px;
```

Comment:
given that `.resources` has `grid` display based on percentages, when the screen size is narrow, the `article` elements are shrinking to less than 100px width...

this is making the images squeezed since sizing of image elements is a little weird. i think you could either fix by changing the height to be `auto` or the `object-fit` of the image to be `contain`

Reply:
Good catch, Seth. I just fixed this issue